benchmarks/src/bin/tpch.rs

```diff
@@ -1106,7 +1106,7 @@ mod ci {
             let actual_row = &actual_vec[i];
             assert_eq!(expected_row.len(), actual_row.len());
 
-            for j in 0..expected.len() {
```

Comment:
e.g., If `expected` contains only one record batch, we only compare first column between rows.

Reply:
Not sure if any existing tests will fail by this.